src/model.py

```diff
@@ -14,8 +14,21 @@ def __init__(self, forward_only, batch_size, use_lstm=True):
         self.batch_size = batch_size
         single_cell = tf.contrib.rnn.GRUCell(config.EMBEDDING_SIZE)
         if use_lstm:
-          single_cell = tf.nn.rnn_cell.BasicLSTMCell(config.EMBEDDING_SIZE)
-        self.cell = tf.contrib.rnn.MultiRNNCell([single_cell for _ in range(config.NUM_LAYERS)])
```

Comment:
[Reference](https://www.tensorflow.org/api_docs/python/tf/nn/rnn_cell/BasicLSTMCell)

Reply:
Yes, we can use LSTMCell instead.